### DISCUSSION TITLE
Automatic Switch?

### DISCUSSION THREAD

**Discussion by rcpalomino** [tryallthethings/DNS-Swapper#8] [Ideas]
a really brilliant extension would be if the little piece of software could recognize if the IP of the PiHole could also be reached and automatically switches there and if not then to the "normal" DNS.

**Comment by tryallthethings** [comment]
Hey @rcpalomino! Sorry for the late reply. 
Is this the kind of functionality you're looking for? 
![image](https://github.com/user-attachments/assets/cc6aa89d-1ebc-457a-8c13-2019c5b49a3f)
